### PR TITLE
November 23 Project Update

### PR DESCRIPTION
**Pull request by lukehinds**
Note, news was gathered with Google News 1 Oct 2023 – 30 Nov 2023

From now on, will limit to a month

### CODE REVIEW

File: content/sigstore-november-roundup-2023.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+author = "Luke Hinds (TSC Chair)"
+type = "post"
+
++++
+
+Welcome to the November edition of the Sigstore Roundup! This is a regular summary of Sigstore news, events, releases and other happenings.
+
+### Sigstore Google Season of Docs 2023 Case Study
+ 
+A very comprehisive case study has been published on the [Sigstore docs wiki](https://github.com/sigstore/docs/wiki/Sigstore-Google-Season-of-Docs-2023-Case-Study) about the Sigstore project's participation in the 2021 program.
```
Comment (bobcallaway) [inline content/sigstore-november-roundup-2023.md:16]:
```suggestion
A very comprehisive case study has been published on the [Sigstore docs wiki](https://github.com/sigstore/docs/wiki/Sigstore-Google-Season-of-Docs-2023-Case-Study) about the Sigstore project's participation in the 2023 program.
```

File: content/sigstore-november-roundup-2023.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+### Latest Releases
+
+#### Rekor v1.3.3
+
+Rekor’s aims to provide an immutable tamper-resistant ledger of metadata generated within a software projects supply chain. Its latest release is [v1.3.3](https://github.com/sigstore/rekor/releases/tag/v1.3.3).
+
+What’s changed:
+
+- Add method to get artifact hash for an entry
```
Comment (bobcallaway) [inline content/sigstore-november-roundup-2023.md:28]: Can we add more items from the changelogs recently? this one is an internal API (not user facing) and reads a bit confusing to me.

File: content/sigstore-november-roundup-2023.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+Keyless Git signing with Sigstore! Its latest release is [v0.8.0](https://github.com/sigstore/gitsign/releases/tag/v0.8.0).
+
+What’s changed:
+
+- Add options for Rekor client, make public key fetcher configurable.
+- Add gitsign initialize. (#321)
+- Fix offline verification marshalling, add e2e tests.
+
+### Sigstore Libraries
```
Comment (bobcallaway) [inline content/sigstore-november-roundup-2023.md:52]: we should also include sigstore-python in this list

File: content/sigstore-november-roundup-2023.md (hunk 2 above)
Comment (lukehinds) [reply content/sigstore-november-roundup-2023.md]: done!

**Review by bobcallaway — APPROVED** [review]

**Pull request merged**